lisa/util/process.py

```diff
@@ -232,6 +246,8 @@ def wait_result(
 
         while self.is_running() and timeout >= timer.elapsed(False):
             time.sleep(0.01)
+            if timer.elapsed(False) > 0.5:
+                self.check_and_input_password()
```

Comment:
If the password is inputed, it shouldn't run again.

Reply:
Yeah, I have added a flag to check.